Mathlib/Topology/Sheaves/Stalks.lean

```diff
@@ -174,26 +172,25 @@ namespace stalkPushforward
 @[simp]
 theorem id (ℱ : X.Presheaf C) (x : X) :
     ℱ.stalkPushforward C (𝟙 X) x = (stalkFunctor C x).map (Pushforward.id ℱ).hom := by
-  -- Porting note: We need to this to help ext tactic.
-  change (_ : colimit _ ⟶ _) = (_ : colimit _ ⟶ _)
-  ext1 j
-  induction' j with j
-  rcases j with ⟨⟨_, _⟩, _⟩
-  erw [colimit.ι_map_assoc]
-  simp [stalkFunctor, stalkPushforward]
+  ext
+  simp only [stalkPushforward, germ, colim_map, ι_colimMap_assoc, whiskerRight_app]
+  erw [CategoryTheory.Functor.map_id]
+  simp [stalkFunctor]
 set_option linter.uppercaseLean3 false in
 #align Top.presheaf.stalk_pushforward.id TopCat.Presheaf.stalkPushforward.id
 
--- This proof is sadly not at all robust:
--- having to use `erw` at all is a bad sign.
 @[simp]
 theorem comp (ℱ : X.Presheaf C) (f : X ⟶ Y) (g : Y ⟶ Z) (x : X) :
     ℱ.stalkPushforward C (f ≫ g) x =
       (f _* ℱ).stalkPushforward C g (f x) ≫ ℱ.stalkPushforward C f x := by
-  change (_ : colimit _ ⟶ _) = (_ : colimit _ ⟶ _)
-  ext U
-  rcases U with ⟨⟨_, _⟩, _⟩
-  simp [stalkFunctor, stalkPushforward]
+  ext
+  simp only [germ, stalkPushforward]
+  -- Now `simp` finishes, but slowly:
```

Comment:
You can't (or don't want to?) combine these two `simp only`s into one `simp only`?

Reply:
I didn't want to. The first shows what we need to unfold, the second is just a plain `simp` called, converted into `simp only` only because it gives a significant speedup.